thanks/thanks.py

```diff
@@ -95,9 +87,51 @@ def get_remote_data(self, project_name):
             data = None
         return data
 
-    def update_give_thanks(self, package_name):
-        if self.debug:
-            print('Checking ', package_name)
-        package_data = self.get_local_data(package_name) or self.get_remote_data(package_name)
-        if package_data:
-            self.give_thanks_to[package_name] = package_data
+    def _generate_output(self, colored_output=True):
+        def _uncolored(text, *args, **kwargs):
+            return text
+
+        colored = termcolor.colored if colored_output else _uncolored
+
+        lines = []
+        lines.append(
+            colored('You depend on {} who would {}'.format(
+                colored('{} authors'.format(len(self.give_thanks_to)), 'cyan'),
+                colored('enjoy donations!', 'green'),
+                ),
+                attrs=['bold']
+            )
+        )
+
+        colorized_data = [
+            ProjectData(
+                name=colored(pd.name, 'cyan'),
+                funding_link=colored(pd.funding_link, 'green'),
+                authors=colored(pd.authors, 'yellow'),
+            )
+            for pd in self.give_thanks_to.values()
+        ]
+
+        lines.append(format_pretty_table(
+            data=colorized_data,
+            column_names=['Project', 'Where to thank', 'Authors'],
+            horizontal_bar=' ',
+            vertical_bar=' ',
+        ))
+
+        lines.append(colored(
+                ''.join([
+                    "If you see projects with ",
+                    colored("MISSING FUNDING INFORMATION ", "red"),
+                    "then why not submit a pull request ",
+                    "the project repository asking the author to ",
+                    colored("ADD A 'FUNDING' PROJECT_URL ", "yellow"),
+                    "to the projects setup.py"
+                ]),
+                attrs=['bold']
+        ))
+
+        return '\n'.join(lines)
+
+    def rocks(self, colored_output=True):
```

Comment:
could we instead call this `give` or `print`? I was confused reading this what `rocks` was referring to.